include/ajax.tasks.php

```diff
@@ -424,7 +422,7 @@ function massProcess($action, $w=null)  {
 
             // Assume success
             if ($i==$count) {
-                $msg = sprintf(__('Successfully %s %s.'),
+                $msg = sprintf(__('Successfully %1$s %2$s.' /* Tokens are <actioned> <x selected task(s)> */),
```

Comment:
is there anyway to test https://github.com/Chefkeks/osTicket-1.8/pull/2 on crowdin? it seems like it should really work.